roles/st2/handlers/main.yml

```diff
@@ -1,6 +1,7 @@
 - name: restart st2
   sudo: true
-  service:
-    name: "{{ item }}"
-    state: restarted
-  with_items: st2_services
+  command: st2ctl restart 
+
+- name: restart st2api
```

Comment:
And while you at it, I'd also ask you to change st2 restart command to reflect https://github.com/StackStorm/st2-packages/pull/285/files